src/chart/line/poly.ts

```diff
@@ -211,12 +211,34 @@ function drawSegment(
     return k;
 }
 
+function getPoints(shape: ECPolygonShape|ECPolylineShape) {
+    let points = Array.from(shape.points);
```

Comment:
Please use `shape.points.slice` instead of `Array.from`. BTW, if `shape.loop` is false, you can just return `shape.points` to improve efficiency.

Reply:
One small thing: the type of `ECPolygonShape|ECPolylineShape` is `ArrayLike<number>` so the `slice` method doesn't exist as it's not a regular array (that's why i used Array.from in the first place). Is it still safe to just force it to be inferred as an array?